sysconfigs/daint/packages.yaml

```diff
@@ -29,6 +29,10 @@ packages:
     bzip2:
         paths:
              bzip2@1.0.6: /usr
+    cce:
+        buildable: false
```

Comment:
I don't know if we have to absolutely want to stop the user to build compilers if they want to